maint-lib/ceph_version.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
Best shebang is `#!/usr/bin/env bash`

Reply:
https://stackoverflow.com/questions/10376206/what-is-the-preferred-bash-shebang/10383546#10383546

Reply:
ok.